dashboard/app.R

```diff
@@ -119,23 +62,23 @@ ui <- fluidPage(padding=0,
             selectInput(
               "forecasters",
               p("Forecasters", tags$br(), tags$span(id="forecaster-input", "Type a name or select from dropdown")),
-              choices = forecasterChoices,
+              choices = c("COVIDhub-baseline", "COVIDhub-ensemble"),
```

Comment:
Is this a constant intended?  It looks like that I add some forecasters, it doesn't add them to the plot:

![image](https://user-images.githubusercontent.com/46794497/112543863-b665e100-8d8c-11eb-9684-e98c49d1401a.png)

Reply:
CMU-TimeSeries does not have forecasts for the US level.